tests/Validation/Constraint/HasClaimTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+declare(strict_types=1);
+
+namespace Lcobucci\JWT\Tests\Validation\Constraint;
+
+use Lcobucci\JWT\Token;
+use Lcobucci\JWT\Validation\Constraint\HasClaim;
+use Lcobucci\JWT\Validation\ConstraintViolation;
+
+/**
+ * @covers \Lcobucci\JWT\Validation\ConstraintViolation
+ * @covers \Lcobucci\JWT\Validation\Constraint\HasClaim
+ *
+ * @uses \Lcobucci\JWT\Token\DataSet
+ * @uses \Lcobucci\JWT\Token\Plain
+ * @uses \Lcobucci\JWT\Token\Signature
+ */
+final class HasClaimTest extends ConstraintTestCase
+{
+    /** @test */
+    public function assertShouldRaiseExceptionWhenClaimIsNotSet(): void
+    {
+        $this->expectException(ConstraintViolation::class);
+        $this->expectExceptionMessage('The token does not have the claim "claimId"');
```

Comment:
Let's put the expectations right before the failing call, please: don't do any other calls in-between, or we'd have hidden regressions. 

This is for all tests

Reply:
I get this format from `HasClaimWithValueTest`. If i change it here to this format - i think it must be done in all tests
```
        $constraint = new HasClaim('claimId');

        $this->expectException(ConstraintViolation::class);
        $this->expectExceptionMessage('The token does not have the claim "claimId"');

        $constraint->assert($this->buildToken());
```

Reply:
You're right that it must done in all tests, we'll eventually get there.

It would be highly appreciated, though, if you could help us by not making the backlog bigger.

Reply:
No problem sir, it's just about consistency between tests :)